mapviz/CMakeLists.txt

```diff
@@ -95,6 +95,12 @@ catkin_package(
 # Fix conflict between Boost signals used by tf and QT signals used by mapviz
 add_definitions(-DQT_NO_KEYWORDS)
 
+# If using <= kinetic, require C++11, otherwise default to C++14
+if ($ENV{ROS_DISTRO} STREQUAL "kinetic" OR $ENV{ROS_DISTRO} STRLESS "kinetic")
+  set(CMAKE_CXX_STANDARD 11)
```

Comment:
Overall this looks okay. However, setting the C++ standard globally like this isn't the best. I would make sure its set per target like before instead.

Reply:
Done. Not sure why those ubuntu focal and debian buster jobs are failing... doesn't look related to the build.